XamarinCommunityToolkit/Views/TimeDiffItem.shared.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.UI.Views
+{
+	struct TimeDiffItem
```

Comment:
Whitespace seems off on the struct name.

Reply:
Oh) Nice catch